examples/lines_bars_and_markers/psd_demo.py

```diff
@@ -58,39 +57,44 @@
 y = y + np.random.randn(*t.shape)
 
 # Plot the raw time series
-fig = plt.figure(constrained_layout=True)
-gs = gridspec.GridSpec(2, 3, figure=fig)
-ax = fig.add_subplot(gs[0, :])
-ax.plot(t, y)
-ax.set_xlabel('time [s]')
-ax.set_ylabel('signal')
+fig, axs = plt.subplot_mosaic([
+    ['signal', 'signal', 'signal'],
+    ['zero padding', 'block size', 'overlap'],
+], constrained_layout=True)
```

Comment:
```suggestion
], layout='constrained')
```
Not that we are doing this everywhere yet, but I think we want to start spelling this like this...